basic-examples/expressions/subworkflow.yaml

```diff
@@ -0,0 +1,20 @@
+version: v0.2.0
+input:
+  root: SubRootObject
+  objects:
+    SubRootObject:
+      id: SubRootObject
+      properties: 
+        name:
+          type:
+            type_id: string
+steps:
+  example:
+    plugin:
+      src: quay.io/arcalot/arcaflow-plugin-template-python

```

Comment:
Examples should use versioned plugins to avoid breakage.

Reply:
One of our core goals is to ensure portability and repeatability of workflows, so version-controlling both plugins and workflows has always been critical to that. Yes, it's possible that a release tag could be retired or deleted, but by our current processes, that does not happen. I think the greater risk is publishing without pinned versions knowing that a :latest tag is almost definitely going to break things in the future. Upward compatibility is definitely not a goal.

Reply:
> Upward compatibility is definitely not a goal.

That's interesting and good to know.  Does that apply more widely than just to workflows and plugins?